src/openforms/formio/tests/test_component_translations.py

```diff
@@ -28,7 +28,10 @@ def disable_prefill_injection():
     """
     Disable prefill to prevent prefill-related queries.
     """
-    return patch("openforms.formio.service.inject_prefill", new=MagicMock)
+    return patch(
+        "openforms.prefill.service.inject_prefill",
+        new=MagicMock,
+    )
```

Comment:
isn't this broken now because of the local import?

Reply:
I don't see a problem here in the tests and if I remove this the tests are failing.

Reply:
I still don't understand why this works but I'll accept it :P